xgcm/grid.py

```diff
@@ -1465,11 +1519,11 @@ def raw_diff_function(data_left, data_right):
 
 
 def raw_min_function(data_left, data_right):
-    return xr.ufuncs.minimum(data_right, data_left)
+    return np.minimum(data_right, data_left)
```

Comment:
silences warnings

Reply:
Ah right. Nice catch. Is our minimum xarray version compatible with using the numpy functions directly? Just checking.